.github/workflows/darwin-tests.yaml

```diff
@@ -101,7 +101,7 @@ jobs:
                   ./scripts/run_in_build_env.sh \
                   "./scripts/tests/run_test_suite.py \
                      --chip-tool ./out/darwin-x64-chip-tool-darwin-${BUILD_VARIANT}/chip-tool-darwin \
-                     --target-skip-glob '{TestGroupMessaging}' \
+                     --target-skip-glob '{TestGroupMessaging,Test_TC_DIAG_TH_NW_1_1,Test_TC_DIAG_TH_NW_1_2}' \
```

Comment:
The right place to disable these for chip-tool-darwin is probably `examples/chip-tool-darwin/templates/tests/tests.js` instead of here.

Reply:
it looks we need to change both places to pass CI

Reply:
Why?  this is going to run "all the tests chip-tool-darwin knows about", and that JS file can be used to change the set of tests it knows about....

Reply:
Because I hit a couple of issues doing this.

After I disable Test_TC_DIAG_TH_NW_1_1 in examples/chip-tool-darwin/templates/tests/tests.js, still failed in Test Suites - Darwin (See the CI failure) 

If I specify Test_TC_DIAG_TH_NW_2_6 ~ Test_TC_DIAG_TH_NW_2_9 in examples/chip-tool-darwin/templates/tests/tests.js, I won't event pass the codegen.

```
🤖 Generating files:
    ✍  /__w/connectedhomeip/connectedhomeip/zzz_generated/chip-tool-darwin/zap-generated/cluster/CHIPTestClustersObjc.h
    ✍  /__w/connectedhomeip/connectedhomeip/zzz_generated/chip-tool-darwin/zap-generated/cluster/CHIPTestClustersObjc.mm
⚠️  Errors:
    👎  /__w/connectedhomeip/connectedhomeip/zzz_generated/chip-tool-darwin/zap-generated/test/Commands.h: ⛔ Error: Test Test_TC_DIAG_TH_NW_2_6  does not exists. [line: 19, column: 22, file: /__w/connectedhomeip/connectedhomeip/examples/chip-tool-darwin/templates/tests/commands.zapt ]
Stack trace:
```